docs/Advanced-Usage/FAQs.rst

```diff
@@ -53,3 +53,25 @@ interactive simulation, or you've connected the NIC to the internet), this is
 likely a bug in one of your custom bridge implementations or in FireSim. In
 fact, for a given target-design, enabling printf synthesis, assertion synthesis,
 autocounter, or Auto ILA, should not change the simulated behavior of the machine.
+
+Is there a way to compress workload results when copying back to the manager instance?
+--------------------------------------------------------------------------------------
+
+FireSim doesn't support compressing workload results before copying them back to the manager instance.
+Instead we recommend that you use a modern filesystem (like ZFS) to provide compression for you.
+For example, if you want to use ZFS to transparently compress data:
+
+#. Attach a new volume to your EC2 instance (either at runtime or during launch).
+   This is where data will be stored in a compressed format.
+#. Make sure that the volume is attached (using something like ``lsblk -f``).
+   This new volume should not have a filesystem type and should be unmounted (named something like ``nvme1n1``).
+#. Install ZFS according to https://zfsonlinux.org/
+#. Mount the volume and setup the ZFS filesystem with compression. 
+
+::
+    zpool create <MOUNTPOINT> /dev/nvme1n1
+    zpool list
+    zfs set compression=gzip <MOUNTPOINT>
```

Comment:
The dataset properties can be specified in one `zpool` command:
```suggestion
    zpool create -o ashift=12 -O compression=on <POOL_NAME> /dev/nvme1n1
    zpool list
    zfs list
```

Also, `compression=on` selects a reasonable default algorithm, usually `lz4`.  `gzip` is usually not recommended as it tends to have a worse trade-off between decompression speed and compression ratio.